tests/test-td-payload/src/testtdinfo.rs

```diff
@@ -27,6 +27,7 @@ pub struct TdInfoRetData {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Tdinfo {
```

Comment:
the name is confusing. Should it be TdTestInfo?

Reply:
The name is not related to this PR. I will ping Wei about it.